packages/assets-controllers/src/TokenBalancesController.ts

```diff
@@ -78,20 +87,9 @@ export type TokenBalancesControllerGetStateAction = ControllerGetStateAction<
   TokenBalancesControllerState
 >;
 
-export type TokenBalancesControllerUpdateChainPollingConfigsAction = {
```

Comment:
It looks like we are removing these actions, should we mention this in the changelog?